### PR TITLE
Fix bow controlling dropped arrow

### LINKED ISSUE #61843: pointless (possibly bugged) check in bow code
image
bruh

### PR DESCRIPTION
**Pull request by PeterMorrison1**
## About The Pull Request
When loading an arrow into a bow and firing the bow without drawing it, the arrow would drop and still be able to be dropped repeatedly, making the arrow follow the player on the ground. See before clip. Also resulted in cleaning up a pointless if statement.

Before clip:

https://user-images.githubusercontent.com/30943236/135733326-c3374db8-3d7a-4149-a99f-ec96f303a4ee.mp4

After clip:

https://user-images.githubusercontent.com/30943236/135733338-7329239f-5fda-4939-9054-d1c78b44ee29.mp4

This also fixes  #61843

## Why It's Good For The Game
Makes the bow work as one would expect it to, if an arrow is on the ground it can't be controlled by the bow until reloaded in the bow.

## Changelog

:cl:
fix: fixed dropped arrows being controllable with the bow
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/bow.dm [hunk 1]
```diff
@@ -23,12 +23,10 @@
...
 	drawn = FALSE
-	if(!chambered)
+	if(chambered)
+		chambered.forceMove(drop_location())
 		chambered = magazine.get_round(keep = FALSE)
-		return
-	if(!chambered)
-		return
-	chambered.forceMove(drop_location())
+		chambered = null
```
Comment (ATH1909) [inline code/modules/projectiles/guns/ballistic/bow.dm:29]: why are you setting chambered to null immediately after setting it to magazine.get_round(keep = FALSE)?

**Comment by ATH1909** [comment]
and thanks for fixing the issue from my issue report!

File: code/modules/projectiles/guns/ballistic/bow.dm (hunk 1 above)
Comment (PeterMorrison1) [reply code/modules/projectiles/guns/ballistic/bow.dm:29]: Ah, I forgot to remove `chambered =`. It just needs the `magazine.get_round(keep = FALSE)` call - thank you for pointing this out. I can update this quickly, should I just commit & push or should I squash the commits? New to public repos :)

**Comment by PeterMorrison1** [comment]
Removed the pointless assignment, my bad :) Let me know if theres anything else I can fix.

**Pull request merged**